### PR TITLE
feedstock "ace_cream" merge request

### PR DESCRIPTION
**Pull request by zhaofeng-shu33**
"ace_cream" is a pypi package which implements ace algorithm, used in non-paramatric estimation.

### CODE REVIEW

File: recipes/ace_cream/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/ace_cream/meta.yaml]: Please delete all comments.

File: recipes/ace_cream/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.  
+  number: 1
```
Comment (synapticarbors) [inline recipes/ace_cream/meta.yaml]: `build/number` should start at `0`.

File: recipes/ace_cream/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+# when copying and paster from another recipe, but not really needed.
+{% set name = "ace_cream" %}
+{% set version = "0.4.post8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/zhaofeng-shu33/ace_cream.git
```
Comment (synapticarbors) [inline recipes/ace_cream/meta.yaml]: Please install from a tarball, not from git:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

File: recipes/ace_cream/build.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+if [ $OSX_ARCH ]; then
+export LDFLAGS=${LDFLAGS}" "-undefined" "dynamic_lookup
+else
+export LDFLAGS=${LDFLAGS}" "-shared
+fi
+$PYTHON setup.py install
```
Comment (synapticarbors) [inline recipes/ace_cream/build.sh]: You should use `pip` as shown in:

https://conda-forge.org/docs/meta.html#use-pip

but adapted to the `build.sh` script. Same goes for `bld.bat`

**Comment by zhaofeng-shu33** [comment]
Hello, since the package requires fortran, I remove the windows build.

File: recipes/ace_cream/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - numpy
```
Comment (synapticarbors) [inline recipes/ace_cream/meta.yaml:22]: Please see the following about building against numpy:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (zhaofeng-shu33) [reply recipes/ace_cream/meta.yaml:22]: > Please see the following about building against numpy:
> 
> https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

I have used `pin_compatible`. However, the build for link still pops up `Package python conflicts`

**Comment by zhaofeng-shu33** [comment]
@conda-forge-admin, please ping team

**Pull request closed without merging**